commands-yml/commands/device/simulator/toggle-touch-id-enrollment.yml

```diff
@@ -0,0 +1,77 @@
+---
+name: Toggle Touch ID Enrollment
+short_description: Toggle the simulator being [enrolled](https://support.apple.com/en-ca/ht201371) to accept touchId  (iOS Simulator only)
+
+description:
+  |
+    To enable this feature, the `allowTouchIdEnroll` desired capability must be set to true. When `allowTouchIdEnroll` is set to true
+    the Simulator will be enrolled by default, and the 'Toggle Touch ID Enrollment' changes the enrollment state.
+
+    This call will only work if Appium process or its parent application (e.g. Terminal.app or Appium.app) has access to Mac OS accessibility in System Preferences > Security & Privacy > Privacy > Accessibility list
+
+example_usage:
+  java:
+    |
+      driver.toggleTouchIDEnrollment(true);
+  python:
+    |
+      self.driver.toggle_touch_id_enrollment()
+  javascript_wd:
+    |
+      await driver.toggleTouchIdEnrollment();
+  javascript_wdio:
+    |
+      driver.toggleTouchIdEnrollment(true);
+  ruby:
+    |
+      @driver.toggle_touch_id_enrollment()
+  php:
+    |
+      // TODO PHP sample
+  csharp:
+    |
+      // TODO C# sample
+
+client_docs:
+  java: "http://appium.github.io/java-client/io/appium/java_client/ios/PerformsTouchID.html#performTouchID-boolean-"
+  python: "https://github.com/appium/python-client/blob/master/appium/webdriver/webdriver.py#L670"
+  javascript_wdio: "http://webdriver.io/api/mobile/toggleTouchIdEnrollment.html"
+  javascript_wd: "https://github.com/admc/wd/blob/master/lib/commands.js#L3150"
+  ruby: "https://github.com/appium/ruby_lib/blob/master/lib/appium_lib/core/common/command.rb#L64"
+  php: "https://github.com/appium/php-client/" # TODO PHP documentation link
+  csharp: "https://github.com/appium/appium-dotnet-driver/" # TODO Dotnet documentation link
+
+# Driver support by platform
+driver_support:
+  ios:
+    xcuitest: true
+    uiautomation: false
+  android:
+    uiautomator2: false
+    uiautomator: false
+  mac:
+    mac: false
+  windows:
+    windows: false
+client_support:
+  java: true
+  python: true
+  ruby: true
+  php: true
+  csharp: true
+  javascript_wd: true
+  javascript_wdio: true
+  
+
+# Information about the HTTP endpoints
+endpoint:
+  url: /session/:session_id/simulator/touch_id
+  method: 'POST'
+  json_parameters:
+    - name: match
```

Comment:
Should be an optional `enabled` parameter. https://github.com/appium/appium-base-driver/blob/master/lib/mjsonwp/routes.js#L427-L429